### ISSUE TITLE
Enable port 32345 protocol dokodemo-door whatever global transparent proxy enable or not

### ISSUE THREAD

**Issue by ZhiShengYuan** [v2rayA/v2rayA#473]
tl;dr This port is in most cases not used by other programs and will have no major adverse effects, but can be easily used with nftables for transparent proxying

In use, I have found that v2raya's web-based management is ideal for Linux devices that act as routers, but since its own transparent proxy modifies iptables (which can be a disaster for some scenarios), so I have to use --lite mode, along with REDSOCKS2, etc., to implement transparent proxies, which can be useful in some scenarios if the port is kept open for inbound traffic whether v2ray's transparent proxy mode is enabled or not

Also, for another reason, v2raya uses iptables to redirect all traffic instead of !chn ip, which may be a better choice for most users, but for me, this operation causes a serious performance loss and unnecessary traffic proxying

**Comment by ZhiShengYuan** [comment]
#395

**Comment by mzz2017** [comment]
--v2ray-confdir may help
